src/annotator/config/test/settings-test.js

```diff
@@ -289,6 +289,29 @@ describe('annotator.config.settingsFrom', function() {
     });
   });
 
+  [
+    {
+      description:
+        "returns an object with the group ID when there's a valid #annotations:group:<ID> fragment",
+      url: 'http://localhost:3000#annotations:group:alphanum3ric_-only',
+      returns: 'alphanum3ric_-only',
```

Comment:
Strictly speaking the character set is "url-safe base 64" (see comments in `annotationsFromURL`), but I think this is fine.